app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  validates :email, presence: true,
+                    format: { with: /\A[^@\s]+@[^@\s]+\z/, message: 'must be a valid address' }
+  validates :first_name, presence: true
+  validates :last_name, presence: true
```

Comment:
This can be put in one line.

```suggestion
  validates :first_name, :last_name, presence: true
```